packages/core/src/navigation/navigation.stories.ts

```diff
@@ -10,7 +10,36 @@ import customElements from '../../dist/core/custom-elements.json';
 import { html } from 'lit';
 import { CdsNavigation } from '@cds/core/navigation';
 import { CdsNavigationGroup } from '@cds/core/navigation';
-import { CdsIcon } from '@cds/core/icon/icon.element.js';
+
+import '@cds/core/icon/register.js';
+import { ClarityIcons } from '@cds/core/icon/icon.service.js';
+
+import {
```

Comment:
I think we can include `ClarityIcons` import here as well.

Reply:
👍. Good catch. Removed.

Reply:
Build fails with the icons used the correct way. Storybook seems to have a hard to trackdown issue with importing _only some_ of the icons. 

I've separated them out for now.  The build is still failing b/c it can't find chromium so I'll try to run it again tomorrow.

Reply:
And now i've come full circle. The isse was not having the filepaths in the import map for the `web-dev-server` config.